README.rst

```diff
@@ -69,6 +69,33 @@ VIA_BASE_URL
   don't have the Hypothesis Chrome extension installed
   (default: https://via.hypothes.is)
 
+Route Syntax/API
+----------------
+
+Share Annotations on Page/URL (``/go``)
++++++++++++++++++++++++++++++++++++++
+
+See the annotations at a URL, and optionally filter or add context to the
+annotations displayed at that URL.
```

Comment:
There are two things which don't come across to me as immediately clear:

1. "See the annotations at a URL" means "go to the URL with Hypothesis embedded" rather than "show me a list of annotations on that URL" (as eg. https://hypothes.is/search?uri={uri} does)
2. "add context to the annotations" means "select which annotations are shown"